negative.py

```diff
@@ -0,0 +1,22 @@
+# Import the dependencies
+import argparse
+import cv2 #image processing lib
+import numpy as np #matrix manipulation
+from time import sleep #for slowing down the process to make progress visible 
+from tqdm import tqdm as tqdm # for progess bar 
+
+#construct the argument parse and parse the arguments
+ap = argparse.ArgumentParser()
+ap.add_argument("-i", "--image", required=True, help="path to input image")
+args = vars(ap.parse_args())
+
+# reading the image
+img = cv2.imread((args["image"]))
+negative = 255 - img
```

Comment:
Try it for pink color(any shade you like) and let me know what we get.

Reply:
Ok